debian/build.go

```diff
@@ -330,10 +326,10 @@ func main() {
 			Distros: serverDistros,
 			Versions: []version{
 				{
-					Version:             "1.6.0-alpha.0.2074-a092d8e0f95f52",
-					Revision:            "00",
-					Channel:             ChannelStable,
-					GetDownloadLinkBase: getCIBuildsDownloadLinkBaseFromVersion,
+					Version:          "1.6.0-alpha.0.2074-a092d8e0f95f52",
+					Revision:         "00",
+					Channel:          ChannelStable,
+					DownloadLinkBase: "https://dl.k8s.io/ci-cross/v1.6.0-alpha.0.2074+a092d8e0f95f52",
```

Comment:
I kinda wonder if instead getCIBuildsDownloadLinkBaseFromVersion should translate the version? I'm a little worried that Version and this URL may end up not matching.